src/components/modal/content/AU/parts/PL.jsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx h */
+import { h } from 'preact';
+import { useContent, useProductMeta } from '../../../lib';
+
+const PL = () => {
```

Comment:
Can we change the name of this file to be GPL.jsx (and wherever it is imported)? Just so it matches the rest of the GPL products rather than GB PL since that's sort of a one-off that we want to change in the future.
```suggestion
const GPL = () => {
```